### ISSUE TITLE
Clean up ibm.ws.wssecurity.fat.untoken.CommonPasswordCallback from com.ibm.ws.wssecurity_fat.wsscxf FAT

### ISSUE THREAD

**Issue by meiaus** [OpenLiberty/open-liberty#15974]
During the testing on new EE8 (jaxws-2.3, etc.) runtime via com.ibm.ws.wssecurity_fat.wsscxf FAT, it's noted this callbackhandler class is in fact not used by either EE7 or EE8 testing, because the server.xml in the server `com.ibm.ws.wssecurity_fat.ssl` has the section with callbackhandler commented out.

Tracking the issue for later cleanup.

```
package com.ibm.ws.wssecurity.fat.untoken;

import java.io.IOException;
import java.util.HashMap;
import java.util.Map;

import javax.security.auth.callback.Callback;
import javax.security.auth.callback.CallbackHandler;
import javax.security.auth.callback.UnsupportedCallbackException;

//orig:
//import org.apache.ws.security.WSPasswordCallback;
//2/2021
import org.apache.wss4j.common.ext.WSPasswordCallback;

/**
 */
public class CommonPasswordCallback implements CallbackHandler {

```
- server.xml
```
	<!-- basicplcy.wssecfvt.test.CommonPasswordCallback -->
	<!-- com.ibm.ws.wssecurity.example.cbh.CommonPasswordCallback -->
	<!-- com.ibm.ws.wssecurity.example.cbh.AnotherPasswordCallback -->
	<!-- <wsSecurityProvider id="default" ws-security.timestamp.timeToLive="120" 
		ws-security.timestamp.futureTimeToLive="60" ws-security.callback-handler="com.ibm.ws.wssecurity.fat.untoken.CommonPasswordCallback" 
		ws-security.cache.config.file="${server.config.dir}/cxf-ehcache.xml"> </wsSecurityProvider> 
		<application name="untoken" location="${server.config.dir}/untoken.war"> 
		<classloader apiTypeVisibility="spec, ibm-api, third-party" /> </application> -->
	<wsSecurityClient
		id="default"
		ws-security.username="user1"
		ws-security.password="security"
	>
	</wsSecurityClient>
```

**Comment by meiaus** [comment]
We'll have another round of update on com.ibm.ws.wssecurity_fat.wsscxf for Jakarta, and will make this cleanup included (target in early May, 2021).

**Comment by meiaus** [comment]
The update is made but will be committed through other FAT update.

**Comment by meiaus** [comment]
Fixed in #17452

**Comment by meiaus** [comment]
#17452 is merged.
